State.lua

```diff
@@ -1066,6 +1066,27 @@ local function removeBuff( aura )
 end
 state.removeBuff = removeBuff
 
+-- Extend buff function, defaults to 1 second if no value supplied, breaks out if supplied a 0
+local function extendBuff( aura, duration)
+
+    if duration == 0 then
+        Error( "Attempted to extend a buff '%s'.\n\n%s by 0 seconds", aura or "nil", debugstack() )
+        return
+    end
+
+    local a = class.auras[ aura ]
+
+    duration = duration or 1
```

Comment:
Does a default of 1 second make sense? Are there places where you see a lot of `expires = expires + 1` in the code already? If not, then maybe better to leave it with no default.

Reply:
Yes, 1 seems to be the most common. Maybe it's better to require a value though? Not sure how I do that.

Reply:
I think requiring the argument is good for documentation purposes at the call location.